### ISSUE TITLE
Corona Light causes an exception

### ISSUE THREAD

**Issue by Remi-Tribia** [BabylonJS/Exporters#677]
Hi,

When there are CoronaLights in the project the exporter throws an exception :

```
Export Cancelled: External component has thrown an exception.
System.Runtime.InteropServices.SEHException (0x800004005): External component has thrown an exception. at Autodesk.Max.Wrappers.LightObject.get_UseAtten() at Max2Babylon.BabylonExporter.ExportLight(IIGameScene scene, IIGameNode lightNode, BabylonScene ...
```

Exporter version is v20190722.2

Cheers!

**Comment by Drigax** [comment]
Looks like Corona is throwing an exception when we try to get the light information. I dont have a license for Corona (partially why #490 is still unaddressed), but I can at the very least add some exception handling logic so that we will instead skip the light or export an empty node when we are given an exception.

**Comment by Drigax** [comment]
At the very least, we should be a bit more robust when we encounter exceptions, and no longer completely crash and burn.

**Comment by Drigax** [comment]
Fix is now available in 20191108.5: https://github.com/BabylonJS/Exporters/releases
